index.html

```diff
@@ -111,6 +111,20 @@ <h2>
         materials</a> are available from the <a href=
         "https://www.w3.org/Consortium/pwe/">Positive Work Environment public homepage</a>.
       </p>
+      <section id="updates">
+        <h3>
+          Updates to CEPC
+        </h3>
+        <p>
+          The CEPC is maintained by the Positive Work Environment Taskforce. In order to keep the CEPC up to date with the needs and scope of W3C, PWE will routinely review and update the CEPC as needed. 
+        </p>
+        <p>
+          In March of each year, the CEPC Review Period will open. During this period PWE will work with W3C members, ombudspeople, and the Advisory Council to intake and review any issues or recommendations for changes to CEPC. This period of review will last 3 months, and should changes be required, PWE will deliver a new version of the CEPC to the AC by the end of July.
+        </p>
+        <p>
```

Comment:
Formal approval of a revised CEPC should require Advisory Board action at a minimum.  The AB might request formal AC review, or even formal AC approval.